docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[cluster-nodes-reload-secure-settings]]
 == Nodes Reload Secure Settings
 
-The cluster nodes reload secure settings API is used to re-read the
-local node's encrypted keystore. Specifically, it will prompt the keystore
-decryption and reading across the cluster. The keystore's plain content is
+The cluster nodes reload secure settings API is used to re-read the local node's encrypted keystore.
```

Comment:
The `local node's encrypted keystore` implies to me the api must be run on every node, when in fact this is a cluster level API, correct? Maybe reword to `...is used to re-load the keystore on each node`? I'm not sure `encrypted` should be there as a qualifier since in the no-passphrase case it is only obfuscated.